### ISSUE TITLE
refactor bcdb [development_bcdbrefactor]

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/jumpscaleX_core#354]
- [x] schema only once per threebot 
- [x] stored in file as data (msgpack)
- [x] huge simplifcation on bcdb level because metadata so much more simple
- [x] more scalable because only loads schema when needed

**Comment by xmonader** [comment]
branch merged into fixes. closing
